src/main/java/org/openrewrite/java/testing/mockito/MockitoWhenOnStaticToMockStatic.java

```diff
@@ -15,26 +15,31 @@
  */
 package org.openrewrite.java.testing.mockito;
 
+import org.jspecify.annotations.Nullable;
 import org.openrewrite.ExecutionContext;
 import org.openrewrite.Preconditions;
 import org.openrewrite.Recipe;
 import org.openrewrite.TreeVisitor;
 import org.openrewrite.internal.ListUtils;
-import org.openrewrite.java.JavaIsoVisitor;
-import org.openrewrite.java.JavaTemplate;
-import org.openrewrite.java.MethodMatcher;
-import org.openrewrite.java.VariableNameUtils;
+import org.openrewrite.java.*;
 import org.openrewrite.java.search.UsesMethod;
-import org.openrewrite.java.tree.Flag;
+import org.openrewrite.java.tree.Expression;
 import org.openrewrite.java.tree.J;
 import org.openrewrite.java.tree.Statement;
 
-import java.util.Collections;
+import java.util.ArrayList;
 import java.util.List;
+import java.util.Optional;
 import java.util.concurrent.atomic.AtomicBoolean;
 
+import static org.openrewrite.java.VariableNameUtils.GenerationStrategy.INCREMENT_NUMBER;
+import static org.openrewrite.java.VariableNameUtils.generateVariableName;
+import static org.openrewrite.java.tree.Flag.Static;
+
 public class MockitoWhenOnStaticToMockStatic extends Recipe {
 
+    private static final AnnotationMatcher BEFORE = new AnnotationMatcher("org.junit.Before");
+    private static final AnnotationMatcher AFTER = new AnnotationMatcher("org.junit.After");
```

Comment:
Like in the [example of this issue](https://github.com/powermock/powermock/issues/1089), I think we should probably also support `@BeforeClass` and `@AfterClass`, right? Although note that the example contains a bug, as it has no corresponding `@AfterClass` annotated method.